### ISSUE TITLE
Allow duplicate preferred labels

### ISSUE THREAD

**Issue by warenhaus** [innoq/iqvoc#336]
iQvoc's consistency check prevents me from publishing a concept that has a *preferred label* which is already in use for another concept.

But especially when *proper names* are concerned, duplicate (or rather multiple) uses of identical names / labels can be correced or even required.

Think for instance of Genealogies: a son may have the exact name as his father/grandfather...
Or place names: an applicaction of **Country->Province->Provincial Captital** would be **Österreich->Salzburg->Salzburg**. 

In short: multiple usage of identical labels can be correct. Admins should have an option to let the consistency check know that the multiple use is ok.

**Comment by mrreynolds** [comment]
@bandholtz What do you think about that?

**Comment by bandholtz** [comment]
when you have duplicate names how will you know which one to pick from a list?
It is good practice to use homonym qualifiers such as in "Salzburg (Province)" and "Salzburg (Capital)".

**Comment by warenhaus** [comment]
thank you for the reply. This is true, though we use the resulting thesaurus in a rather unsusal way: as support for full-text search, which means that labels like "Salzburg (Province)" create search terms that are not helpful. 

Can you point me to the piece(s) of code where the check for duplicates is performed during the consistency check? I might be able to see if we can deactivate it on a more hack-basis.

**Comment by accuosto** [comment]
It is frequent to have thesauri with duplicate labels in different parts of the hierarchy.
This limits the possibilities of using iqVoc in many real cases.

**Comment by warenhaus** [comment]
Please allow me to come back to this with another argument pro duplicate preflabels: Isn't a digital thesaurus a good way to indicate (in a way readable both for machines and humans) exactly what concept is referred to by some label? So if something is described as or linked to the thesaurus' concept "Österreich->Salzburg->Salzburg", it is perfectly identifiable as the City of Salzburg, while something linked to "Österreich->Salzburg" is understood as referring to the province of Salzburg.

tl/dr: using a digital thesaurus to annotate stuff _enables_ the use of duplicate labels - it should not prevent it.
